src/Metadata/Extractor/YamlExtractor.php

```diff
@@ -32,7 +32,7 @@
     protected function extractPath(string $path)
     {
         try {
-            $resourcesYaml = Yaml::parse(file_get_contents($path));
+            $resourcesYaml = Yaml::parse(file_get_contents($path), Yaml::PARSE_KEYS_AS_STRINGS);
```

Comment:
I have deprecation warnings from the component without this change and tests fail...